pkg/skullking/round.go

```diff
@@ -37,7 +46,8 @@ func (t *Trick) Play(p Play) error {
 
 // Winner will return the player that wins the current Trick
 func (t *Trick) Winner() *Player {
-	return nil
+	winnerPosition := t.Winner2()
```

Comment:
I don't think winner2 is a good name for a function.

Reply:
Winner2 was meant only as a WIP, now it's called WinnerPosition